src/components/ListDetail/ListDetail.module.css

```diff
@@ -167,49 +174,58 @@
 
 /**
 * List detail number
-* 1) Used on list items using number variant
+* 1) Used on list items using number variant; provides a cosmetic cicular border
+* 2) Using flex to keep the number centered
+* 3) Since this will display as a cirlce, keep height and width equal; should be wide enough to fit a two-digit number
+* 4) Using a percentage here instead of --eds-theme-border-radius because this is not a typical bordered object
 */
 .list-detail__number {
   position: relative;
+  display: flex; /* 2 */
+  justify-content: center;
+  align-items: center;
   left: -4px;
-  width: 12px;
-  height: 12px;
-  border-radius: 50%;
-  border: 1px solid var(--eds-theme-color-neutral-subtle-border);
+  width: 1.5rem; /* 3 */ 
+  height: 1.5rem; /* 3 */
+  border-radius: 50%; /* 4 */
```

Comment:
couldn't this still be `eds-border-radius-round`? or should we avoid it b/c that's a tier-1 token definition?

Reply:
I personally like the idea of decoupling this from the border token simply because this value needs to remain unchanged here.